src/components/Approval/Approval.tsx

```diff
@@ -62,7 +62,7 @@ export const Approval = () => {
   } = useFormContext()
 
   const activeQuote = useSwapperStore(selectQuote)
-  const feeAssetFiatRate = useSwapperStore(selectFeeAssetFiatRate)
+  const feeAssetFiatRate = useSwapperStore(selectFeeAssetUserCurrencyRate)
```

Comment:
seeing quite a few variable names that weren't updated with the same vernacular if we want to keep that in alignment

Reply:
Think I got most in the commits following this comment, hard to cover it all but at least the main ones should be tackled here - we can probably tackle the rest as we see them